### PR TITLE
[BIOMAGE-1009] Fix error sns messages

### PR DESCRIPTION
**Pull request by cosa65**
# Background
#### Link to issue 
https://biomage.atlassian.net/browse/BIOMAGE-1009

#### Link to staging deployment URL 
https://ui-martinfosco-ui289-api136-p.scp-staging.biomage.net/

#### Links to any Pull Requests related to this
https://github.com/biomage-ltd/ui/pull/289
https://github.com/biomage-ltd/pipeline/pull/73

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: src/specs/models/GEM2SResponse.v1.yaml [hunk 1]
```diff
@@ -1,22 +1,30 @@
-title: Pipeline response
+title: GEM2S response
```
Comment (aerlaut) [inline src/specs/models/GEM2SResponse.v1.yaml:1]: Can you check this schema again? It doesn't seem right. `experimentId` is type `string` but has `taskName` as it's attribute according to the indentation.

File: src/specs/models/PipelineResponse.v1.yaml [hunk 2]
```diff
@@ -2,42 +2,40 @@ title: Pipeline response
...
+            type: string
+            description: The name of the task that was executed.
+          config:
+            $ref: ./ProcessingConfigBodies.v1.yaml
+        required:
+          - experimentId
+          - taskName
+      required:
+        - output
+        - response
```
Comment (aerlaut) [inline src/specs/models/PipelineResponse.v1.yaml]: the key `response` doesn't exist in this schema

File: src/specs/models/PipelineResponse.v1.yaml [hunk 3]
```diff
@@ -2,42 +2,40 @@ title: Pipeline response
...
+          - bucket
+          - key
+        properties:
+          bucket:
+            type: string
+            description: The S3 bucket where the response body is located.
+          key:
+            type: string
+            description: The key within the S3 bucket where the result is located.
+      input:
```
Comment (aerlaut) [inline src/specs/models/PipelineResponse.v1.yaml]: is input a property of `output`? or is it an option under `oneOf`?

File: tests/api/route-services/__snapshots__/pipeline-response.test.js.snap [hunk 4]
```diff
@@ -12,7 +12,7 @@ exports[`Test Pipeline Response Service functions propely with correct input (cu
         },
         "Key": Object {
           "experimentId": Object {
-            "S": "1234",
+            "S": undefined,
```
Comment (aerlaut) [inline tests/api/route-services/__snapshots__/pipeline-response.test.js.snap]: According to the test, the value of `experimentId` is passed in the test, while it was previously passed. Are you sure this is the expected behaviour?

File: tests/api/route-services/__snapshots__/pipeline-response.test.js.snap [hunk 5]
```diff
@@ -61,11 +61,7 @@ exports[`Test Pipeline Response Service functions propely with correct input (cu
             },
           },
         },
-        "Key": Object {
-          "experimentId": Object {
-            "S": "1234",
-          },
-        },
+        "Key": Object {},
```
Comment (aerlaut) [inline tests/api/route-services/__snapshots__/pipeline-response.test.js.snap]: Same as above.

**Review by aerlaut — CHANGES_REQUESTED** [review]
Can you check the API.yaml definitions and the tests again?

File: src/specs/models/PipelineResponse.v1.yaml (hunk 3 above)
Comment (cosa65) [reply src/specs/models/PipelineResponse.v1.yaml]: it is an option under oneOf

File: tests/api/route-services/__snapshots__/pipeline-response.test.js.snap (hunk 4 above)
Comment (cosa65) [reply tests/api/route-services/__snapshots__/pipeline-response.test.js.snap]: You're right, I forgot to adapt the fake message to the new structure received, adding an experimentId where it is taken from now fixes it.

**Review by aerlaut — APPROVED** [review]
LGTM

**Pull request merged**